include/text.php

```diff
@@ -608,17 +527,9 @@ function logger($msg, $level = 0) {
 	$a = get_app();
 	global $LOGGER_LEVELS;
 
-	// turn off logger in install mode
```

Comment:
How do we now ensure to not continue in install mode?

Reply:
The `App->mode` now is a bitfield with multiple capabilities flags. From the mode you can now check that the local config file is present, that the DB is accessible, and that the config table is present. In particular the `Config` class uses these flags to decide whether to look for the config in DB or just retrieve from the `App->config` array.